torchdynamo/optimizations/distributed.py

```diff
@@ -63,6 +63,10 @@ def compile_fn(self, gm: fx.GraphModule, example_inputs: List[torch.Tensor]):
                 )
                 bucket_bytes += params_size_b
                 # print(f"accumulated {params_size_b} b from {node}")
+            elif node.op == "get_attr":
+                maybe_param = getattr(gm, node.target)
```

Comment:
cool, i think theoretically this would also catch buffers (which we generally also want for DDP -- in case we didn't pass synchronize_buffers=False) , but it will also catch other tensors that happen to be attributes of an nn.Module even if they are neither parameters or buffers. I think that's ok .

Reply:
hmm, what are 'buffers' used for and is there any other way to identify them? i'm guessing we don't have that information if we've captured them out of a custom layer that's been traced through, so this may be the best we can do.